### ISSUE TITLE
Test: `accessibilityHelpContent` view property API

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#43130]
Refs -209855

- [ ] Windows 
- [ ] macOS 
- [ ] Linux 

Complexity: 3

---

Accessibility help dialogs exist around the workbench - in the terminal, chat input, editor, and more. With focus in these areas, in screen reader mode, you can view them by pressing `alt+f1`. 

Extensions, like the GHPR extension, want to contribute help dialogs. There is proposed API to allow adding this to a view via `accessibilityHelpContent`. 

Add this for a view. When your focus is in the view, use `alt+f1` and you should see the content that you've provided. 

Remember to include: `"contribAccessibilityHelpContent"` in your `"enabledApiProposals"`. Verify that it does not work if you don't include this.

https://github-com/microsoft/vscode/blob/7b49653d3a5e47284e72b6a858ed1a90cda25141/src/vs/workbench/api/browser/viewsExtensionPoint.ts-L174
